example-app/src/services/NasaService.ts

```diff
@@ -77,9 +77,30 @@ class NasaService {
         id: itemData.nasa_id,
         title: itemData.title,
         link: itemLinks.href
-      }
-    })
-  );
+      };
+    });
+
+  getAstronomyPictureData = async (): Promise<
+    Global.AstronomyPictureDataShape
+  > => {
+    const astronomyPictureDataUrl = `https://api.nasa.gov/planetary/apod?api_key=${this.apiKey}`;
+
+    let body = await this.smartFetch(astronomyPictureDataUrl);
```

Comment:
remove spaces before and after
it should be declared with "const" instead of "let"